app/javascript/packs/submit-with-spinner.js

```diff
@@ -0,0 +1,8 @@
+import { loadPolyfills } from '@18f/identity-polyfill';
+
+loadPolyfills(['element-closest']).then(() => {
```

Comment:
From testing, I found that technically `element-closest` is already available in older browsers, because it's polyfilled by USWDS (more specifically, its `receptor` dependency):

https://github.com/shawnbot/receptor/blob/2ed497c/src/delegate.js#L1-L2

It seems like it'd be an easy thing to break if we try to rely on this indirect dependency, though. Hence loading it explicitly here, even if duplicated.